tests/CMakeLists.txt

```diff
@@ -63,7 +66,9 @@ add_subdirectory(unit_tests)
 add_subdirectory(difficulty)
 add_subdirectory(hash)
 add_subdirectory(net_load_tests)
-add_subdirectory(libwallet_api_tests)
+
+# Disabled until issue #895 is resolved
+#add_subdirectory(libwallet_api_tests)
```

Comment:
All the documentation I've seen put the coveralls reporting in the after_success step which is not reached if any tests fail. I will experiment and see if there are any issues reporting coverage for a failing test suite.